beta-src/src/models/BoardClass.ts

```diff
@@ -0,0 +1,83 @@
+import { IBoard } from "./Interfaces";
+import TerritoryClass from "./TerritoryClass";
+import UnitClass from "./UnitClass";
+
+// this is will be used for accessing all necessary data and methods
+
+export default class BoardClass {
+  // all territories.
+  territories: TerritoryClass[];
+
+  // includes entire units currently placed in the board.
+  units: UnitClass[];
+
+  // This will be used for order generation.
+  myUnits: UnitClass[];
```

Comment:
who is my here? i know you copied this over, but is this the player? maybe we can call it playerUnits?